### ISSUE TITLE
Getting started guide code raises error

### ISSUE THREAD

**Issue by mikeyford** [pymc-devs/pymc#3333]
I was following the [getting started guide](https://docs.pymc.io/notebooks/getting_started.html) and right at the start, the code raises a seemingly basic error.

**Please provide a minimal, self-contained, and reproducible example.**
```
>>> import pymc3 as pm
>>> print('Running on PyMC3 v{}'.format(pm.__version__))
Running on PyMC3 v3.6
>>> basic_model = pm.Model()
>>> with basic_model:
...     alpha = pm.Normal('alpha', mu=0, sigma=10)
...     beta = pm.Normal('beta', mu=0, sigma=10, shape=2)
...     sigma = pm.HalfNormal('sigma', sigma=1)
...     mu = alpha + beta[0]*X1 + beta[1]*X2
...     Y_obs = pm.Normal('Y_obs', mu=mu, sigma=sigma, observed=Y)
```

**Please provide the full traceback.**
```Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
  File "/Users/mike/anaconda3/lib/python3.6/site-packages/pymc3/distributions/distribution.py", line 41, in __new__
    dist = cls.dist(*args, **kwargs)
  File "/Users/mike/anaconda3/lib/python3.6/site-packages/pymc3/distributions/distribution.py", line 52, in dist
    dist.__init__(*args, **kwargs)
  File "/Users/mike/anaconda3/lib/python3.6/site-packages/pymc3/distributions/continuous.py", line 437, in __init__
    super(Normal, self).__init__(**kwargs)
  File "/Users/mike/anaconda3/lib/python3.6/site-packages/pymc3/distributions/distribution.py", line 180, in __init__
    shape, dtype, defaults=defaults, *args, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'sigma'
```

**Please provide any additional information below.**

It seems from the help that the sigma keyword can be changed to `sd`. Would that be the correct usage? If so, it would be great to update the tutorial for the latest version.

## Versions and main components

* PyMC3 Version: 3.6
* Theano Version: 1.0.3
* Python Version: 3.6
* Operating system: OSX 10.13.6
* How did you install PyMC3: conda

**Comment by fonnesbeck** [comment]
Looks like you are trying to run an example from master using a release version of PyMC3.

**Comment by fonnesbeck** [comment]
Yes, it appears that our docs website is from current master rather than the current release, which should not be the case. cc @twiecki @ColCarroll

**Comment by Nathan-Furnal** [comment]
An easy fix is to replace the argument `sigma` by `sd` in the functions argument when writing the code. @mikeyford 

```
basic_model = pm.Model()

with basic_model:

    # Priors for unknown model parameters
    alpha = pm.Normal('alpha', mu=0, sd=10)
    beta = pm.Normal('beta', mu=0, sd=10, shape=2)
    sigma = pm.HalfNormal('sigma', sd=1)

    # Expected value of outcome
    mu = alpha + beta[0]*X1 + beta[1]*X2

    # Likelihood (sampling distribution) of observations
    Y_obs = pm.Normal('Y_obs', mu=mu, sd=sigma, observed=Y)
```

**Comment by Zhanyu-Wang** [comment]
From the source code of PyMC3 v3.6, we should use something like 
`pm.Normal('theta_i', mu=param_mu, sd=param_tau)`

**Comment by alando46** [comment]
@fonnesbeck @twiecki hey guys really excited to check out project. just fyi I'm still getting this error when trying to follow documentation on the 3.6 build (installed with conda). Is there something I'm missing?

[Edit] Fixed the issue :) Note for others: Version 3.7 installed via `conda install -c conda-forge pymc3` fixes this. the version of pymc3 installed by `conda install pymc3` for some reason is 3.6 on my machine.
